tests/components/homeassistant/test_init.py

```diff
@@ -136,10 +136,14 @@ def tearDown(self) -> None:
 
     def test_is_on(self):
         """Test is_on method."""
-        assert comps.is_on(self.hass, "light.Bowl")
-        assert not comps.is_on(self.hass, "light.Ceiling")
-        assert comps.is_on(self.hass)
-        assert not comps.is_on(self.hass, "non_existing.entity")
+        with pytest.raises(
+            RuntimeError,
+            match="Detected code that uses homeassistant.components.is_on. This is deprecated and will stop working",
+        ):
+            assert comps.is_on(self.hass, "light.Bowl")
+            assert not comps.is_on(self.hass, "light.Ceiling")
+            assert comps.is_on(self.hass)
+            assert not comps.is_on(self.hass, "non_existing.entity")
```

Comment:
Sorry to be a pain but won't everything but the first one be skipped now?

Reply:
Yea, indeed, we can remove then